src/physics.F90

```diff
@@ -465,6 +472,12 @@ subroutine elastic_scatter(i_nuclide, rxn, E, uvw, mu_lab, wgt)
     ! Set energy and direction of particle in LAB frame
     uvw = v_n / vel
 
+    ! Because of floating-point roundoff, it may be possible for mu_lab to be
+    ! outside of the range [-1,1). In these cases, we just set mu_lab to exactly
+    ! -1 or 1
+
+    if (abs(mu_lab) > ONE) mu_lab = sign(ONE,mu_lab)
```

Comment:
Aren't all of these checks in elastic_scatter, sab_scatter, and inelastic_scatter redundant since scatter also has a check that mu is in the proper range?